content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst

```diff
@@ -148,47 +149,57 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
+resulting :guilabel:`Work Orders` pop-up window, click the :guilabel:`Open Shop Floor` button to
+open the *Shop Floor* module.
 
-.. image:: measure_check/measure-tablet-view.png
+When accessed from a specific work order, the *Shop Floor* module opens to the page for the work
+center where the order is configured to be processed, and isolates the work order's card so that no
+other cards are shown.
+
+Process the work order's steps until the *Measure* quality check step is reached. Click on the step
+to open a pop-up window that includes instructions for how the measurement should be taken. After
+taking the measurement, enter it in the :guilabel:`Measure` field of the pop-up window, and then
+click :guilabel:`Validate`.
+
+.. image:: measure_check/measure-check-shop-floor.png
    :align: center
-   :alt: A Measure quality check in the Manufacturing tablet view.
+   :alt: A Measure check in the Shop Floor module.
 
 If the measurement entered is within the range specified in the :guilabel:`Tolerance` section of the
-|QCP|, the quality check passes and the tablet view moves on to the next step of the work order.
-However, if the measurement entered is outside of the specified range, a pop-up window appears,
+|QCP|, the quality check passes, and the pop-up window moves on to the next step of the work order.
+However, if the measurement entered is outside of the specified range, a new pop-up window appears,
 titled :guilabel:`Quality Check Failed`.
 
-The body of the :guilabel:`Quality Check Failed` pop-up window shows a warning message that states,
-:guilabel:`You measured # units and it should be between # units and # units.`, as well as the
+The body of the :guilabel:`Quality Check Failed` pop-up window shows a message that states,
+:guilabel:`You measured # units and it should be between # units and # units`, as well as the
 instructions entered in the :guilabel:`Message If Failure` tab of the |QCP|. At the bottom of the
-pop-up, two buttons appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
+pop-up window, two buttons appear: :guilabel:`Correct Measure` and :guilabel:`Confirm Measure`.
 
-.. image:: measure_check/measure-check-failed.png
+.. image:: measure_check/shop-floor-measure-check-failed.png
    :align: center
-   :alt: The "Quality Check Failed" pop-up window.
+   :alt: The Quality Check Failed pop-up window for a Measure check in the Shop Floor module.
 
 If the measurement was not entered correctly, and should be changed, select :guilabel:`Correct
 Measure`. Doing so opens a new pop-up window, titled :guilabel:`Quality Check`. Enter the corrected
-measurement in the :guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check,
-and move on to the next step of the work order.
+measure in the :guilabel:`Measure` field, and then click :guilabel:`Validate` to complete the check
+and close the pop-up window.
```

Comment:
```suggestion
measure in the :guilabel:`Measure` field, then click :guilabel:`Validate` to complete the check and
close the pop-up window.
```

Reply:
I've been told to include the "and" in cases like this.